honeypotio/website
# Kb rn/version locks

## Summary
The purpose os this PR is to use the correction version of node, as specified in the `.nvmrc` file, when installing dependencies and running the website, both locally and in the build pipeline

## How to Test
- Trigger a build
- On the TravisCI logs you should see:
   `Found '/home/travis/build/honeypotio/website/.nvmrc' with version <10.16.3>`
   `Downloading and installing node v10.16.3...`
   `Downloading https://nodejs.org/dist/v10.16.3/node-v10.16.3-linux-x64.tar.xz...`
   `Computing checksum with sha256sum`
   `Checksums matched!`
   `Now using node v10.16.3 (npm v6.9.0)`
